src/scene_battle_rpg2k3.cpp

```diff
@@ -301,7 +301,7 @@ void Scene_Battle_Rpg2k3::CreateUi() {
 	}
 
 	if (lcf::Data::battlecommands.battle_type != lcf::rpg::BattleCommands::BattleType_traditional) {
-		int transp = lcf::Data::battlecommands.transparency == lcf::rpg::BattleCommands::Transparency_transparent ? 128 : 255;
+		int transp = lcf::Data::battlecommands.transparency == lcf::rpg::BattleCommands::Transparency_transparent ? 160 : 255;
```

Comment:
btw. this is also referenced in multiple places, so maybe make this a function as well? `…IsTransparent()`
https://github.com/EasyRPG/Player/search?q=battlecommands.transparency

Reply:
good idea, as this is 2k3 only feature the  `IsTransparent`` func can be in the 2k3 battle scene directly.